contracts/modules/claims/ClaimsReward.sol

```diff
@@ -529,10 +528,25 @@ contract ClaimsReward is IClaimsReward, LegacyMasterAware {
     }
     lengthVote = cd.getVoteAddressMemberLength(msg.sender);
     lastClaimed = lengthVote;
-    _days = _days.mul(counter);
-    if (tc.tokensLockedAtTime(msg.sender, "CLA", now) > 0) {
-      tc.reduceLock(msg.sender, "CLA", _days);
+
+    {
+      uint lockTimePerVote = td.lockCADays();
+      uint reduction = lockTimePerVote.mul(counter);
+      uint minExpiration = block.timestamp.add(lockTimePerVote);
+      uint currentExpiration = tc.getLockedTokensValidity(msg.sender, "CLA");
+
+      // reduce lock time only if current lock time is greater than min lock time
+      if (currentExpiration > minExpiration) {
+
+        // and never reduce to less than min lock time
+        if (currentExpiration.sub(reduction) < minExpiration) {
```

Comment:
so this means that you will always have to wait at least min lock time correct?

There's no way to get this lock time down to block.timestamp